css/style.css

```diff
@@ -16,4 +19,144 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/*Header*/
+
+header {
+  display: flex;
+  padding: 0 5rem;
+  justify-content: space-between;
+}
+
+/*Karma Logo*/
+
+header img {
+  height: 3rem;
+  width: 3rem;
+  margin-top: 10px;
+}
+
+/*Navigation*/
+
+ul {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+}
+
+li {
+  padding: 0.5rem;
+}
+
+a {
+  color: gray;
+  text-decoration: none;
+}
+
+a:hover {
+  color: var(--orange-karma);
+}
+
+/*Intro Image*/
+
+.intro {
+  display: flex;
+  background-image: url("../img/first-background.jpg");
+  padding: 15.5rem 0;
+  color: white;
+  flex-direction: column;
+  align-items: center;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  margin-bottom: 6rem;
+}
+
+/*Intro Text*/
+
+h1,
+h2,
+h3,
+.intro p {
+  font-weight: lighter;
+  text-align: center;
+}
+
+h1 {
+  font-size: 2.5rem;
+  margin: 0;
+}
+
+.intro p {
+  font-size: 1.5rem;
+}
 
+/*Intro Button*/
+
+button {
+  padding: 0.75rem 1.25rem;
+  background-color: #f26639;
```

Comment:
Missed this one var, but in general, good work on adding variables to your CSS :tada:

Reply:
Corrected missing var.